src/evo/deterministicmns.cpp

```diff
@@ -945,7 +945,8 @@ void CDeterministicMNManager::DecreasePoSePenalties(CDeterministicMNList& mnList
     // only iterate and decrease for valid ones (not PoSe banned yet)
     // if a MN ever reaches the maximum, it stays in PoSe banned state until revived
     mnList.ForEachMN(true, [&](const CDeterministicMNCPtr& dmn) {
```

Comment:
maybe
```suggestion
    mnList.ForEachMN(true /* onlyValid */, [&](const CDeterministicMNCPtr& dmn) {
```
instead of (or in addition to) the comment below?

Reply:
I thought of that, but my IDE adds that in for me :P